### DISCUSSION TITLE
Proces one job at time

### DISCUSSION THREAD

**Discussion by Zikoel** [taskforcesh/bullmq#413] [Q&A] [answered]
First, Thank you for this `masterpiece`! I very like bull and bullmq!

In my use-case have some very heavy (CPU and memory) and slow jobs! I want to be sure that only one job per worker is executing! What is in your opinion the best way to do that with this library. Exploring the docs I only see a temporary rate limiter not `how many job per worker` limiter! I will try with some tests for sure but I want an opinion from more expert users.

**Comment by manast** [comment, marked as answer]
Just make sure that you are only running 1 worker with concurrency factor 1 at any moment in time.

**Reply by KIRANSHANKARK** [reply]
I need to process 1 job at a time in bullmq and the concurrency is set to 1 for that worker...but it does not working!

**Reply by Zikoel** [reply]
What means `not working` ? What is your behaviour? Remember that the concurrency option act only on that worker! If you have 10 workers every worker will process only a job at time, so 10 job at time in total!

**Reply by KIRANSHANKARK** [reply]
The job  is some sort of manipulation of the incoming transaction and based on that we will get a remote port and ip address in which we send those processed 
or validated transaction.the issue is when multiple transactions hit the worker,it is processing the transactions in parallel.I need to process them one by one. concurrency of the worker is set to 1.but it didn't work.do you have any solution? Kindly share...


Thank you.
